Time off tracker WEB/appsettings.Development.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    }
+  }
+  /*"ConnectionStrings": {
```

Comment:
don't commit commented code, uncomment it. It's ok to commit db connection in case of development config, not production one